package.json

```diff
@@ -9,6 +9,9 @@
     "web": "expo start --web"
   },
   "dependencies": {
+    "@react-navigation/stack": "^6.3.17",
+    "@rneui/base": "^4.0.0-rc.7",
```

Comment:
It's better to use stable versions. Your library here is a release candidate (rc) which might have bugs.